### DISCUSSION TITLE
Analyst Client Rendering Issues

### DISCUSSION THREAD

**Discussion by dannitco** [TIBCOSoftware/spotfire-mods#115] [Q&A]
I am working on a Mod that won’t render in analyst, but will render in the browser on the same machine when I use copy url to clipboard from developer tools. This exact same mod renders in analyst client & browser just fine on a different machine. I get the following error message in the console:
“Your WebGL implementation doesn’t seem to support hardware accelerated rendering. Check your browser settings or if your GPU is in a blocklist.” I did the following already: checked all the chrome settings to make sure the browser does support hardware acceleration, I used the webgl checker to make sure the browser supports webgl, and made sure there’s no GPU blocklist. I also have hardware acceleration turned on in Spotfire and set the default browser to chrome. Any ideas? I thought to change the AWS instance type to something that has GPU graphics support, but the instance type of the machine where the mod renders in analyst doesn’t have any GPU, so not sure if that would help. Any ideas appreciated! Thank you

**Comment by objerke** [comment]
Hi @dannitco, could you tell which version of Spotfire Analyst you've encountered this issue? I know that there has been a hotfix of Spotfire 11.4 last year, regarding WebGL, which could be related.

[TS-69547 - Custom JavaScript code, including visualization mods, does not work due to lack of WebGL support.](https://community.tibco.com/s/article/List-of-hotfixes-for-TIBCO-Spotfire-Clients-Analyst-Web-Player-Consumer-Business-Author-and-Automation-Services#1140)

**Reply by dannitco** [reply]
hi there- Thanks for your response. The machine where It renders correctly is 11.6 and the machine where I am having the issue is 12.2

**Reply by objerke** [reply]
@dannitco Would you mind testing if the issue persists in Spotfire 12.4 which is the latest release?

**Reply by dannitco** [reply]
Yes I will test

**Reply by dannitco** [reply]
It still did not render in 12.4. We are working in 12.2 but  I tried the same mod in Spotfire 11.6 and it worked fine.

**Reply by objerke** [reply]
@dannitco could you share a small mod code sample that reproduces the issue? If you can't share the code I think you will need to reach out to the Spotfire support team for further investigation.

**Reply by dannitco** [reply]
Thank you Olaf- I can send the code but I do not think there is an issue with the code. I believe it is on the spotfire side. Also the data is quite specific.  Is it an option to spend 10 mins on a quick call internally to review? Thank you for your help
